app/components/battleship.hbs

```diff
@@ -0,0 +1,46 @@
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+ width="240" height="140" viewBox="0 0 1024.000000 541.000000"
+ preserveAspectRatio="xMidYMid meet" class="battleship" ...attributes>
+  <g transform="translate(0.000000,541.000000) scale(0.100000,-0.100000)"
+stroke="none">
+    <path d="M4397 5328 c-4 -46 -7 -186 -7 -313 0 -126 -4 -235 -8 -241 -4 -6
```

Comment:
how come you have this inline instead of using the asset you added

Reply:
You can't change the fill color with CSS otherwise.

Reply:
Tried a few different ways, but this was the best way to keep it encapsulated and allow CSS based fill coloring.